### ISSUE TITLE
We would like to edit subblocks, which basically is a block in an attribute value

### ISSUE THREAD

**Issue by sebasmannem** [minamijoyo/hcledit#67]

**Comment by sebasmannem** [comment]
Like in a file with contents like:
```
module "sg" {
  source        = "git.git@git"
  cloud_region  = var.cloud_region
  global_config = var.global_config
  custom_name   = local.identifier
  vpd_id        = var.vpc_id
  ingress_rules = {
    https = {
      service = "psql-db"
      network = "seb_test"
    }
  }
}
module "sa" {
  source        = "git.git@git"
  cloud_region  = var.cloud_region
  global_config = var.global_config
  custom_name   = local.identifier
  vpd_id        = var.vpc_id
  ingress_rules = {
    https = {
      service = "psql-db"
      network = "seb_test"
    }
  }
}
```
we would like to edit the network attribute in the https attribute in the ingress_rules attribute in module "sg".

**Comment by minamijoyo** [comment]
Closed this as duplicate of #28. See also my comment at https://github.com/minamijoyo/hcledit/pull/68#issuecomment-1453282366
